src/lib/x509/pkix_enums.h

```diff
@@ -11,6 +11,8 @@
 
```

Comment:
```suggestion
#include <string>
```

Appeasing the MSVC build.